v5/v5_stable.ino

```diff
@@ -1922,4 +1966,38 @@ void updateDebugger(int index, int value) {
     Println("Updating value of : " + varName + " to : " + String(value));
     updateSPIFFS(varName, String(value));
   }
+}
+
+void updateMQTT(int temperature_, int humidity_) {
+  if (!client.connected()) {
+    reconnect();
+  }
+  client.loop();
+  client.publish("v1/devices/me/telemetry",
+                 ("{temperature:" + String(temperature_) +
+                  ",humidity:" + String(humidity_) + "}")
+                     .c_str());
+  Println(4, "MQTT updated");
+}
+
+void toOrangePi(String str) { println("{hay orange-pi! " + str + "}"); }
+
+String askOrangPi(String str) {
+  toOrangePi(str + "?");
+  wait(500);
+  return replyOfOrangePi();
+}
+
+String replyOfOrangePi() {
+  unsigned int startTime = millis() / 1000;
+  String reply = "";
+  while (
+      (!(reply.indexOf("hay ttgo-tcall!") != -1 && reply.indexOf("}") != -1)) &&
+      (millis() / 1000) - startTime < 10) {
+    if (SerialMon.available()) {
+      reply += SerialMon.readString();
+    }
+  }
+  Println(5, "Reply of Orange Pi : " + reply); // TODO: orange-pi debugger
```

Comment:
According to PR rules you should complete your work to approve it, so please write a debugger code for orange pi communication and implement it completely other debuggers are working.